src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  const bestDiscount = 50;
+  const standardDiscount = 20;
```

Comment:
The names 'bestDiscount' and 'standardDiscount' could be more descriptive. Consider naming them to reflect the conditions under which they apply, such as 'LONG_TERM_DISCOUNT' and 'SHORT_TERM_DISCOUNT'.